Fix lsp_options for lspconfig[server.name].setup(lsp_options), close #19

hmgle opened the pull request.

hmgle committed:
Fix lsp_options for lspconfig[server.name].setup(lsp_options), close #19

AlixShahid reviewed (changes requested):
looks great. But can you please 
1. change the name of variable ```opt``` to ```server_options```
2. add new line between ```for loop``` and ```local opt={}``` (line 172-173)
3.  re-add newline to last line (241)

hmgle commented:
> looks great. But can you please
> 
>     1. change the name of variable `opt` to `server_options`
> 
>     2. add new line between `for loop` and `local opt={}` (line 172-173)
> 
>     3. re-add newline to last line (241)

No problem, I will update.

hmgle committed:
[nvim-lspconfig] update

1. add new line between for loop and local opt={}
2. change the name of variable opt to server_options
3. re-add newline to last line

AlixShahid reviewed (approved):
thanks
